frontend/packages/akr/src/pages/PrivacyPolicyPage.tsx

```diff
@@ -111,297 +107,175 @@ export const PrivacyPolicyPage = () => {
+              <br /> {translatePrivacy('dataContents.group2.action')}
             </Text>
+            <Text>{translatePrivacy('dataContents.group2.description')}</Text>
             <Text>
-              <b>Ryhmä 3</b>: Rekisteriin merkityt auktorisoidut kääntäjät
+              <b>{translatePrivacy('common.group')} 3</b>:{' '}
+              {translatePrivacy('dataContents.group3.name')}
+              <br /> {translatePrivacy('dataContents.group3.action')}
             </Text>
             <Text>
-              Kääntäjiä auktorisoidaan kolmella perusteella:
+              {translatePrivacy('dataContents.group3.grounds.description1')}:
               <BulletList
-                items={[
-                  'hyväksytysti suoritettu auktorisoidun kääntäjän tutkinto',
-                  'maisterin tutkinnossa suoritettu kääntämisen korkeakouluopinnot (60 opintopistettä, johon sisältyy  auktorisoidun kääntämisen 6 opintopistettä)',
-                  'virallinen kääntäjä',
+                localisationKeys={[
+                  'dataContents.group3.grounds.ground1',
+                  'dataContents.group3.grounds.ground2',
+                  'dataContents.group3.grounds.ground3',
                 ]}
               />
+              {translatePrivacy('dataContents.group3.grounds.description2')}
             </Text>
             <Text>
-              Kaksi ensin mainittua uusivat auktorisoidun kääntäjän oikeutensa 5
-              vuoden välein. Viralliselle kääntäjälle myönnetään auktorisointi
-              ilman takarajaa. Opetushallitus lähettää rekisteristä muistutuksen
-              3 kk ennen auktorisoinnin umpeutumista. Jos auktorisointi
-              umpeutuu, poistetaan kääntäjä julkisesta hakukoneesta. Kääntäjä
-              voi milloin tahansa hakea auktorisointia uudelleen.
-            </Text>
-            <Text>
-              Rekisteröitävän antamat tiedot
+              {translatePrivacy('dataContents.group3.details.heading')}
               <br />
-              Henkilötiedot: nimi, henkilötunnus, osoite, sähköpostiosoite,
-              puhelinnumero ja asuinpaikka.
+              {translatePrivacy('dataContents.group3.details.description')}
             </Text>
             <Text>
-              Muut tiedot:
-              <br />
+              {translatePrivacy(
+                'dataContents.group3.details.otherDetails.description'
+              )}
+              :
               <BulletList
-                items={[
-                  'auktorisointiperuste',
-                  'mistä kielestä mihin kieleen kääntäjä on auktorisoitu tekemään käännöksiä',
+                localisationKeys={[
+                  // TODO: new swedish document contains 4 details
+                  'dataContents.group3.details.otherDetails.detail1',
+                  'dataContents.group3.details.otherDetails.detail2',
+                  // TODO: there should be some extra descriptions below above receivers.heading
```

Comment:
Tää hanskataan vissiin sitten siinä vaiheessa kun ruvetaan läiskimään niitä ruotsinkielisiä käännöksiä?

Reply:
Joo, joko silloin tai jos saadaan nyt weeklyssä ratkaisu tähän / uusin suomenkielinen versio jossa nää mahdollisesti on, niin vois tänkin PR:n yhteydessä laittaa. Mut ihan kummin päin vaan.